### ISSUE TITLE
login/logout not logged on /var/log/wtmp; 'last ' does not show when users log in or out

### ISSUE THREAD

**Issue by lmarengo** [sddm/sddm#748]
SDDM does not seem to log when a user logs in or out into /var/log/wtmp. Console logins are logged properly, but X logins are not. Here's a snippet of the output of 'last'; there were four different users logged in during that period.

```
reboot   system boot  4.4.27-2-default Sun Nov 27 12:33 - 00:22  (11:49)     
Marcos   pts/0        :0               Sat Nov 26 20:51 - 20:57  (00:05)     
reboot   system boot  4.4.27-2-default Sat Nov 26 12:56 - 00:22 (1+11:26)    
reboot   system boot  4.4.27-2-default Fri Nov 25 10:26 - 00:22 (2+13:56)    
reboot   system boot  4.4.27-2-default Thu Nov 24 21:09 - 00:22 (3+03:13)    
reboot   system boot  4.4.27-2-default Thu Nov 24 17:28 - 20:11  (02:43)     
reboot   system boot  4.4.27-2-default Thu Nov 24 15:36 - 20:11  (04:35)     
reboot   system boot  4.4.27-2-default Thu Nov 24 12:00 - 20:11  (08:11)     
reboot   system boot  4.4.27-2-default Thu Nov 24 09:26 - 09:48  (00:21)     
papi     pts/0        192.168.76.27    Wed Nov 23 18:29 - 18:30  (00:00)     
reboot   system boot  4.4.27-2-default Wed Nov 23 16:12 - 09:48  (17:36)     
reboot   system boot  4.4.27-2-default Wed Nov 23 12:22 - 12:39  (00:17)     
reboot   system boot  4.4.27-2-default Tue Nov 22 18:05 - 12:39  (18:34)     
reboot   system boot  4.4.27-2-default Tue Nov 22 12:06 - 13:59  (01:52)     
reboot   system boot  4.4.27-2-default Tue Nov 22 07:04 - 07:35  (00:30)
```

**Comment by mafn** [comment]
This seems to be related to the missing utmp support (see the TODO page in the wiki). Are there plans to implement it or can this be offloaded to other tools, e.g. logind?

**Comment by lmarengo** [comment]
The issue appears to be resolved as of 0.17. Closing bug.
